build/flaws/broken-links.js

```diff
@@ -34,6 +34,31 @@ function isHomepageURL(url) {
   return split.length === 3 && VALID_LOCALES.has(split[1].toLowerCase());
 }
 
+function mutateLink(
+  $element,
+  { suggestion, enUSFallback, isSelfLink } = {
+    suggestion: null,
+    enUSFallback: null,
+    isSelfLink: false,
+  }
+) {
+  if (isSelfLink) {
+    $element.addClass("self-link");
```

Comment:
I am wondering whether we should perhaps use `aria-current="true"` instead of a CSS class here. It is nice for accessibility and we can also use it as a style hook. I know there are other instances, like in the updates to sidebars, where we also want to indicate that an entry is a link to the current page.

Would be nice if we consistently used the ARIA state for all these instances. We can then even move this styling into minimalist so, it essentially does not matter whether it ends up inside document content or the sidebar, if it has `aria-current="true"` it will receive the same styling.

Thoughts @hamishwillee @peterbe?

Reply:
@schalkneethling Looking at the docs there are two reasonable values for this:
- page: current page within a set of pages.
- true: current item within a set.

Generally we are looking at links in a set of pages for the "Learn" modes and "Related topics".
I would say that a sidebar or breadcrumb would use "page". You would use "true" for any other "set" of items. So my _guess_  is that  "page" is more appropriate and that it would make great sense for that particularly use case.

For things like the `RTCIdentityEvent` this probably doesn't technically apply as the self links are not part of a set. 

The problem here is that I'm not an expert on aria. So I could be completely wrong on the above and I don't know the "cost" if we apply page/true to a link that is outside a set. Any expert we could ask (unless that is you)?

Reply:
Those are good points and this phrase "in a set" did bother me a little. It is one of those specification phrases I see a lot that is really open to interpretation 😄  The link in question in this issue could be argued to be part of a set of links in the main content but, I agree we need more input here so we do no harm.

I am still reading my way through the ARIA spec and would not consider myself an expert, yet :) I do have some I can ask though that totally _is_ an expert on this.

Reply:
If you can be more specific than "true", then use "page". For aria-current, if something has a specific value, it should in my opinion take precedence over the very generic "true" value. So yeah, I think "page" is OK in this case.

Reply:
Brilliant! Thanks so much @MarcoZehe

Reply:
So it should be `<a href="/current/url" aria-current="page">` when it's a regular link within the page. Good. 
What if it's a bullet point list in a sidebar? Is it still `page` that counts or should it be `true` there?

Reply:
Yes, it should be `page`. Sidebars are the perfect case of "if it can be more specific than "true", then use "page" in https://github.com/mdn/yari/pull/3866#discussion_r641422929